k-distribution/tests/regression-new/checks/checkIsSort.k

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) K Team. All Rights Reserved.
+module CHECKISSORT-SYNTAX
+  imports BOOL-SYNTAX
+  imports BASIC-K
+
+  syntax Bool ::= isNonAddr(KItem) [function]
+  syntax NonAddr ::= nonAddr(KItem)
```

Comment:
Can you also add a declaration that is correct?
This way we can test for a positive case.